source/azure_iot_hub_client.c

```diff
@@ -665,7 +665,7 @@ AzureIoTHubClientResult_t AzureIoTHubClient_SetSymmetricKey( AzureIoTHubClient_t
         ( pucSymmetricKey == NULL ) || ( ulSymmetricKeyLength == 0 ) ||
         ( xHMACFunction == NULL ) )
     {
-        AZLogError( ( "AzureIoTHubClient_SetSymmetricKey failed: Invalid argument" ) );
+        AZLogError( ( "AzureIoTHubClient_SetSymmetricKey failed: invalid argument" ) );
```

Comment:
>" [](start = 81, length = 1)

I see full stop at the end of comment and some places not. can we be consistent with that 

Line:698

Reply:
Freertos looks like it doesn't so I'll remove them